data/raw/keybindings.json

```diff
@@ -2620,13 +2620,6 @@
     "name": "Select items @ all 9 fields",
     "bindings": [ { "input_method": "keyboard", "key": "a" }, { "input_method": "keyboard", "key": "A" } ]
   },
-  {
-    "type": "keybinding",
-    "id": "ITEMS_DRAGGED_CONTAINER",
-    "category": "ADVANCED_INVENTORY",
-    "name": "Select items in dragged container",
-    "bindings": [ { "input_method": "keyboard", "key": "d" }, { "input_method": "keyboard", "key": "D" } ]
-  },
```

Comment:
Seems like you removed the wrong keybinding (the `ITEMS_CONTAINER` action is the one below).